tests/test_linalg.py

```diff
@@ -115,3 +115,74 @@ def test_matrix_rank():
             result = inp.linalg.matrix_rank(ia, tol=tol)
             expected = numpy.linalg.matrix_rank(a, tol=tol)
             numpy.testing.assert_array_equal(expected, result)
+
+
+def test_norm1():
+    arrays = [
+        [7],
+        [1, 2],
+        [1, 0]
+    ]
+    ords = [None, -numpy.Inf, -2, -1, 0, 1, 2, 3, numpy.Inf]
+    axis_ = [0, None]
+    for array in arrays:
+        for ord in ords:
+            for axis in axis_:
+                a = numpy.array(array)
+                ia = inp.array(a)
+                print('a = ', a)
+                print('axis = ', axis)
+                print('ord = ', ord)
+                result = inp.linalg.norm(ia, ord=ord, axis=axis)
+                print('result = ', result)
+                expected = numpy.linalg.norm(a, ord=ord, axis=axis)
+                print('expected = ', expected)
+                numpy.testing.assert_allclose(expected, result)
+
+
+# def test_norm2():
+#     arrays = [
+#         [[1, 0]],
+#         [[1, 2]],
+#         [[1, 0], [3, 0]],
+#         [[1, 2], [3, 4]]
+#     ]
+#     ords = [None, -numpy.Inf, -2, -1, 1, 2, numpy.Inf, 'fro', 'nuc']
+#     axis_ = [(0, 1), None]
+#     keepdims = [True, False]
+#     for array in arrays:
+#         for ord in ords:
+#             for axis in axis_:
```

Comment:
Please, remove multiline comments and add disabled tests to `skipped_tests_gpu.tbl` and/or `skipped_tests.tbl`. Or just simply remove unnecessary code, if it is needed.